.github/workflows/build_and_deploy.yml

```diff
@@ -40,6 +38,12 @@ jobs:
         id: update-tags
         run: git fetch --tags -f
 
+      - name: 'Fetch merge commit'
+        id: set-merge-commit
+        run: |
+          merge_commit=$(git rev-parse :/"^Merge pull request")
```

Comment:
This is just getting the most recent merge commit?

Reply:
What the git command does is `returns the youngest matching commit which is reachable from any ref, including HEAD.`

So it will return the first merge commit with the commit message it finds before the tag that launched the deploy job.

Given that tagging is automatic by versionbot on merging PRs into master, and the deploy job only triggers if the tag matches a specific pattern, I think it's safe.